espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/model/SourceDocument.kt

```diff
@@ -198,8 +199,8 @@ class SourceDocument @JvmOverloads constructor(
     }
 
     private fun performCleanup() {
-        if (tmpXmlName != null) {
-            getApplicationContext<Context>().deleteFile(tmpXmlName)
+        tmpXmlName?.let {
+            getApplicationContext<Context>().deleteFile(it)
             tmpXmlName = null
```

Comment:
shouldn't this be out of the let block?

Reply:
It re-assigns it to `null` after it's done with it.